R/cluster.R

```diff
@@ -156,13 +157,15 @@ makeCluster <-
 
     # log into private registry if registry credentials were provided
     if (!is.null(config$dockerAuthentication) &&
-        !is.null(config$dockerAuthentication$username)) {
+        !is.null(config$dockerAuthentication$username) &&
+        nchar(config$dockerAuthentication$username) > 0 &&
+        nchar(config$dockerAuthentication$password) > 0 &&
```

Comment:
We have is.null check for username, do we need is.null on password and registry as well?

Reply:
I think so because they could in theory fill in any one of the three values and it will fail unless they are all filled out.